closed/test/jdk/openj9/internal/security/TestConstraintsSuccess.java

```diff
@@ -95,7 +96,7 @@ public void runWithConstraints() throws Exception {
         outputAnalyzer.shouldHaveExitValue(0);
     }
 
-    public static void main(String[] args) throws Exception {
+    public static void main(String[] args) throws Throwable {
```

Comment:
This doesn't need to change: the throws clause of `getInstances()` didn't change.